### PR TITLE
add dymension playground testnet

### PR DESCRIPTION
**Pull request by maxzonder**

### CODE REVIEW

**Review by dylanschultzie — APPROVED** [review]

**Comment by dylanschultzie** [comment]
This LGTM, @JeremyParish69 can you review if you have time?

File: testnets/dymensionplaygroundtestnet/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,164 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "dymensionplaygroundtestnet",
```
Comment (JeremyParish69) [inline testnets/dymensionplaygroundtestnet/assetlist.json:3]: this should just be 'dymensiontestnet', not have 'playground' in there.

File: testnets/dymensionplaygroundtestnet/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+      "type_asset": "sdk.coin",
+      "base": "adym",
+      "name": "Dymension",
+      "display": "dym",
+      "symbol": "DYM",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/dymensionplaygroundtestnet/images/dymension-logo.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/dymensionplaygroundtestnet/images/dymension-logo.svg"
+      },
+      "coingecko_id": "dymension",
```
Comment (JeremyParish69) [inline testnets/dymensionplaygroundtestnet/assetlist.json:26]: We aren't adding CGIDs to testnet tokens.

File: testnets/dymensionplaygroundtestnet/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/dymensionplaygroundtestnet/images/dymension-logo.png",
```
Comment (JeremyParish69) [inline testnets/dymensionplaygroundtestnet/assetlist.json:29]: These images aren't identical...
And there already exists dymension logos in the registry, so we don't need duplicates.

File: testnets/dymensionplaygroundtestnet/assetlist.json [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+      "type_asset": "ics20",
+      "base": "ibc/FB53D1684F155CBB86D9CE917807E42B59209EBE3AD3A92E15EF66586C073942",
+      "name": "Nim Network",
+      "display": "nim",
+      "symbol": "NIM",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "nim",
```
Comment (JeremyParish69) [inline testnets/dymensionplaygroundtestnet/assetlist.json:66]: Usually mainnet tokens aren't being sent to a testnet. This likely comes from nimtestnet.

File: testnets/dymensionplaygroundtestnet/assetlist.json [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+      "type_asset": "ics20",
+      "base": "ibc/5A26C8DC8DA66F4DD94326E67F94510188F5F7AFE2DB3933A0C823670E56EABF",
+      "name": "Mande Network",
+      "display": "mand",
+      "symbol": "MAND",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "mande",
```
Comment (JeremyParish69) [inline testnets/dymensionplaygroundtestnet/assetlist.json:127]: Same issue here--likely not from mainnet mande

**Comment by JeremyParish69** [comment]
I think we should revert this change and submit again once the issues are corrected

**Comment by dylanschultzie** [comment]
thanks for the second set of eyes, @JeremyParish69

**Comment by maxzonder** [comment]
thanks @JeremyParish69 

re: dymensionplaygroundtestnet - dymension runs 2 testnets, blumbus and playground, that's where the name comes from

re: nim and mande - on playground testnet, these tokens have exactly the same ibc denom and path as mainnet tokens

re: images and coingecko - i can fix it with PR

what do you think? cc @dylanschultzie

**Comment by JeremyParish69** [comment]
> re: dymensionplaygroundtestnet - dymension runs 2 testnets, blumbus and playground, that's where the name comes from
> 
When there are multiple simultaneous testnets, we append an incrementing integer: 'dymensiontestnet1', dymentiontestnet2', ..., etc. The name "playground" can be retained in the pretty_name and chain_id.

> re: nim and mande - on playground testnet, these tokens have exactly the same ibc denom and path as mainnet tokens
> 
It's fine for the ibc denoms to happen to match, but the chain_names referenced here are necessarily mainnet names. Assuming they actually come from a testnet, it should instead say 'mandetestnet' and 'nimtestnet' (however they're registered)

> re: images and coingecko - i can fix it with PR
> 
This would be great

> what do you think? cc @dylanschultzie

**Pull request merged**